spec/index.bs

```diff
@@ -13485,8 +13491,7 @@ and {{GPUTextureUsage/STORAGE_BINDING|GPUTextureUsage.STORAGE_BINDING}} usage re
     <tr>
         <td>{{GPUTextureFormat/rg11b10ufloat}}
         <td>{{GPUTextureSampleType/"float"}},<br/>{{GPUTextureSampleType/"unfilterable-float"}}
-        <td><!-- Vulkan -->
-        <td><!-- no multisampling without RENDER_ATTACHMENT (gpuweb/gpuweb#2465) -->
```

Comment:
(aside: I'm so glad we put this comment here or I wouldn't have even thought about making it support multisampling.)